### PR TITLE
docs/faq improvements…

### PR DESCRIPTION
**Pull request by michaelstingl**
… based on https://github.com/owncloud/ios-app/pull/781

## Description

FAQ page improvements based on feedback in https://github.com/owncloud/ios-app/pull/781

## How Has This Been Tested?

```
git clone https://github.com/owncloud/ios-app.git
cd ios-app
git checkout docs/improvements+faq
cd docs
yarn install
yarn antora
yarn serve
```

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by mmattel** [comment]
There is missing content for section `== Feature Requests`.
Please either create one or remove that section.
(see my comment in referenced merged PR)

File: docs/modules/ROOT/pages/ios_faq.adoc [hunk 1]
```diff
@@ -10,12 +13,12 @@ Here you can find some of the most frequently asked questions about the ownCloud
...
 
 Previous ownCloud iOS app with support for ownCloud server < 10.0 is still available in App Store:  
-https://apps.apple.com/app/id543672169
+{ios-legacy-appstore-url}
 
 === Compatibility with iOS versions older than iOS 12.0
 
 Previous ownCloud iOS app with support for iOS < 12.0 is still available in App Store:  
-https://apps.apple.com/app/id543672169
+{ios-legacy-appstore-url}
```
Comment (mmattel) [inline docs/modules/ROOT/pages/ios_faq.adoc:21]: I cant do an easy commit-proposal as it spawns two lines here.
Proposing the following:
`Previous ownCloud iOS app with support for iOS < 12.0 is still available in {ios-legacy-appstore-url}[App Store]`
This makes the link part of the sentence and better readable.

File: docs/modules/ROOT/pages/ios_faq.adoc [hunk 2]
```diff
@@ -33,17 +36,17 @@ https://github.com/owncloud/ios-app/issues/604
...
-Here you can find previous discussion:
+Here you can find previous discussions:
 https://github.com/owncloud/ios-app/issues/317
 
-=== Translation missing or bug
+=== Missing translations or translation bugs
 
 You can help translate in your language or fix a bug.
 
 We use Transifex for translations. Please register for an account and join the global community:
```
Comment (mmattel) [inline docs/modules/ROOT/pages/ios_faq.adoc:46]: Proposing:
`We use Transifex for translations. Please {ios-app-tx-url}[register here] for an account and join the global community.`

File: docs/modules/ROOT/pages/ios_faq.adoc [hunk 3]
```diff
@@ -33,17 +36,17 @@ https://github.com/owncloud/ios-app/issues/604
...
-https://www.transifex.com/owncloud-org/owncloud/mobile-ios-app/
+{ios-app-tx-url}
 
 === Testing new features
 
 Join the beta program on iOS:
```
Comment (mmattel) [inline docs/modules/ROOT/pages/ios_faq.adoc:51]: Proposing:
`Use {ios-app-beta-url}[this link] to join the beta program on iOS.`

File: docs/modules/ROOT/pages/ios_faq.adoc (hunk 1 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_faq.adoc]: fixed with https://github.com/owncloud/ios-app/pull/798/commits/28783c88091138713ac017492fccccda7e21dfd6

File: docs/modules/ROOT/pages/ios_faq.adoc (hunk 2 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_faq.adoc]: fixed with https://github.com/owncloud/ios-app/pull/798/commits/28783c88091138713ac017492fccccda7e21dfd6

File: docs/modules/ROOT/pages/ios_faq.adoc (hunk 3 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_faq.adoc]: fixed with https://github.com/owncloud/ios-app/pull/798/commits/28783c88091138713ac017492fccccda7e21dfd6

**Pull request merged**